pkg/runtime/hotreload/loader/operator/component.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	compapi "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
+	operatorpb "github.com/dapr/dapr/pkg/proto/operator/v1"
+	"github.com/dapr/dapr/pkg/runtime/hotreload/loader"
+)
+
+type component struct {
+	operatorpb.Operator_ComponentUpdateClient
+}
+
+//nolint:unused
```

Comment:
Why these methods are included if unused?

Reply:
The go linter we are using hasn't learned (maybe never will) that these functions are being used by the generic loader and so complains.

Reply:
Ok. Can you please add a comment explaining this?